JUNIE.md

```diff
@@ -0,0 +1,68 @@
+# Junie Project Guidelines
```

Comment:
@jetbrains-junie I think the file should be `.junie/guidelines.md`. Create junie/guidelines.md file with the following information:
* The coding conventions being followed in the current codebase
* The code organization and package structure 
* Unit and integration testing approaches being followed